### PR TITLE
*Works* WebSocket fixes for big endian processors

### PR DESCRIPTION
**Pull request by smipi1**

### CODE REVIEW

File: include/boost/beast/websocket/detail/frame.hpp [hunk 1]
```diff
@@ -272,7 +273,7 @@ read_close(
     {
         std::uint8_t b[2];
         buffer_copy(buffer(b), cb);
-        cr.code = big_uint16_to_native(&b[0]);
+        cr.code = (::new(&b[0]) big_uint16_buf_t)->value();
```
Comment (vinniefalco) [inline include/boost/beast/websocket/detail/frame.hpp]: is this a separate bug?

Comment (smipi1) [reply include/boost/beast/websocket/detail/frame.hpp]: Oops... Yes. Sorry.

Comment (vinniefalco) [reply include/boost/beast/websocket/detail/frame.hpp]: if you could please submit that in a separate pull request I will add it to 156

Comment (vinniefalco) [reply include/boost/beast/websocket/detail/frame.hpp]: This causes test failures and is generally not correct, it clobbers the value in `b`

**Comment by vinniefalco** [comment]
The is resolved in the latest version

**Pull request closed without merging**